.travis.yml

```diff
@@ -31,35 +31,42 @@ env:
 matrix:
   include:
     # Linux {
-    - os: linux
-      env: PROJECT_DIR=examples/foo TOOLCHAIN=default
-    - os: linux
-      env: PROJECT_DIR=examples/foo TOOLCHAIN=clang-libstdcxx
-    - os: linux
-      env: PROJECT_DIR=examples/foo TOOLCHAIN=gcc-5
-    - os: linux
-      env: PROJECT_DIR=examples/foo TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon
-    - os: linux
-      env: PROJECT_DIR=examples/foo TOOLCHAIN=android-ndk-r15c-api-21-armeabi-v7a-neon-clang-libcxx
-    - os: linux
-      env: PROJECT_DIR=examples/foo TOOLCHAIN=analyze
-    - os: linux
-      env: PROJECT_DIR=examples/foo TOOLCHAIN=sanitize-address
-    - os: linux
-      env: PROJECT_DIR=examples/foo TOOLCHAIN=sanitize-leak
-    - os: linux
-      env: PROJECT_DIR=examples/foo TOOLCHAIN=sanitize-thread
+#     - os: linux
+#       env: PROJECT_DIR=examples/foo TOOLCHAIN=default
```

Comment:
`s,foo,FunctionalPlus,g`: https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#fetch-ci-configs

Reply:
Sorry about that.  I did that initially, then removed it since they didn't seem relevant to C++14 (couldn't be fixed in a follow up effort), then botched the PR when adding it back.